docs/install/configurations.rst

```diff
@@ -0,0 +1,341 @@
+
+::
+
+    # .env
+    SITE_WARNING_BANNER_ENABLED=True
+    SITE_NAME="My site"
+
+    # systemd config file
+    [Service]
+    Environment="SITE_WARNING_BANNER_ENABLED=True"
+    Environment="SITE_NAME=My site"
+
+will enable the warning banner and define the name of the site as "My site". Note that the variables are namespaced: ``SITE_FOO=BAR`` for variables concerning the general configuration, ``ZGW_BAR=BAZ`` for variables concerning the configuration of ZGW, and so on. For an overview of the features that support automatic configuration and the relevant environment variables, see ``Supported configurations`` below.
+
+
+Usage
+=====
+
+If the project is being configured for the first time, run the command from the project root:
+
+::
+
+    src/manage.py setup_configuration
+
+
+By default, ``setup_configuration`` checks if a configuration already exists and will stop executing if it finds one. In order to overwrite an existing configuration, use:
+
+::
+
+    src/manage.py setup_configuration --overwrite
+
+
+Also by default, ``setup_configuration`` tests the configuration to detect problems. You can disable this with the following:
+
+::
+
+    src/manage.py setup_configuration --no-selftest
+
+
+For a full overview of the command and its options:
+
+::
+
+    src/manage.py setup_configuration --help
+
+
+
+Supported configurations
+========================
+
+Below are the features and settings that can be configured via environment variables. For a more detailed overview, run the following command:
+
+::
+
+    src/manage.py config_info <CONFIGURATION>
+
+where `CONFIGURATION` is one of "siteconfig", "kic", "zgw", "digid_oidc", "eherkenning_oidc".
+
+
+In the following, ``FOO=<value1 | value2 | value3>`` means that only ``FOO=value1``, ``FOO=value2``, and ``FOO=value3`` are admissible.
+
+
+General configuration
+^^^^^^^^^^^^^^^^^^^^^
+
+Required:
+"""""""""
+
+::
+
+    SITE_PRIMARY_COLOR
+    SITE_SECONDARY_COLOR
+    SITE_ACCENT_COLOR
+    SITE_PRIMARY_FONT_COLOR
+
+
+All variables:
+""""""""""""""
+
+::
+
+    SITE_CONFIG_ENABLE=True
+    SITE_NAME="My site"
+    SITE_SECONDARY_COLOR="#000000"
+    SITE_ACCENT_COLOR="#000000"
+    SITE_PRIMARY_FONT_COLOR="#111111"
+    SITE_SECONDARY_FONT_COLOR="#222222"
+    SITE_ACCENT_FONT_COLOR="#333333"
+    SITE_WARNING_BANNER_ENABLED=<True | False>
+    SITE_WARNING_BANNER_TEXT="warning banner text"
+    SITE_WARNING_BANNER_BACKGROUND_COLOR="#444444"
+    SITE_WARNING_BANNER_FONT_COLOR="#555555"
+    SITE_LOGIN_SHOW=False
+    SITE_LOGIN_ALLOW_REGISTRATION=<True | False>
+    SITE_LOGIN_2FA_SMS=<True | False>
```

Comment:
These lists of values should be sorted so it more readable and related values are grouped together. Also I miss the meta information like type, description, default etc.

Reply:
I added more information as part of the refactoring. Not sure how to approach the sorting. I agree the order could be improved, I don't see an easy way to automate this.

Reply:
I'd assume if you can print a list of things you could sort them before the printing? I put some notes of possible locations.